assets/styles/main.css

```
@@ -0,0 +1,168 @@
+body {
+  overflow-x: hidden;
+}
+
+main {
+  text-align: left;
+  margin: 0 auto;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+.flex-h,
+.flex-v {
+  display: flex;
+}
+
+.flex-v {
+  flex-direction: column;
+}
+
+.flex-h {
+  flex-direction: row;
+}
+
+.page-top {
+  height: 530px;
+  background-image: url(../images/your_verse_hero.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: 0 100%;
+  padding-top: 70px;
+  flex-basis: 100%;
+}
+
+.promos {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+}
+
+.promos a {
+  width: 25%;
+}
+
+footer div {
+  display: flex;
+  align-content: space-between;
+  margin-left: 10%;
+  margin-right: 10%;
+  font-size: small;
+}
+
+.promos a div {
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 200px;
+}
+
+.promos-img1 {
+  background-image: url(../images/promo_iphone5s.jpg);
+}
+
+.promos-img2 {
+  background-image: url(../images/promo_iphone5c.jpg);
+}
+
+.promos-img3 {
+  background-image: url(../images/promo_30_years.jpg);
+}
+
+.promos-img4 {
+  background-image: url(../images/promo_supplier_responsibility.jpg);
+}
+
+footer ul {
+  list-style: none;
+  padding-right: 5px;
+}
+
+.page-top nav ul {
+  max-width: 900px;
+  height: 40px;
+  margin: 0 auto;
+  border: 1px solid #707070;
+  background-image: linear-gradient(#777 0%, #777 50%, #666 50%, #666 60%, #777 100%);
+  align-items: center;
+  list-style-type: none;
+  align-content: space-between;
+  border-radius: 5px;
+}
+
+footer ul li {
+  padding-right: 5px;
+}
+
+.page-top nav ul li {
+  flex: 1;
+  text-align: center;
+  justify-content: center;
+  border-right: 1px solid #707070;
+}
+
+.page-top nav ul li:last-child {
+  border: 0;
+}
+
+.page-top ul a {
+  text-decoration: none;
+  color: white;
+}
```

Please do it like the original sample, so please change the anchor tag colour to blue. 
```suggestion
.page-top ul a {
  text-decoration: none;
  color: blue;
}
```